docs/usage/security-and-permissions.md

```diff
@@ -65,6 +65,101 @@ For example, if you have an `npm` package and do not configure a private registr
 You could avoid this by configuring private registries but such registries need to query public registries anyway.
 We don't know of any public registries which reverse lookup IP addresses to associate companies with packages.
 
+#### Security advisory for self-hosted Renovate instances
+
+##### Introduction
+
+Before you start self-hosting Renovate you must understand the security implications associated with monitoring and updating repositories.
+The process that Renovate uses to update dependencies runs under the same user context as the Renovate process itself.
+This means it (what does it refer to here???) has the same level of access to information and resources as the user context!
```

Comment:
it refers to "process that Renovate uses to update dependencies"

Reply:
```suggestion
This also means the process has the same level of access to information and resources as the user context!
```